sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -117,11 +117,13 @@ def error(self, message):
 
 
 class PipelineOptions(HasDisplayData):
-  """Pipeline options class used as container for command line options.
+  """Subclasses of this class are used as containers for command line options.
```

Comment:
nit: This class is also used as a container.

Reply:
Fair point. Updated.